mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -297,13 +315,15 @@ class RFormula @Since("1.5.0") (@Since("1.5.0") override val uid: String)
   @Since("1.5.0")
   // optimistic schema; does not contain any ML attributes
   override def transformSchema(schema: StructType): StructType = {
-    require(!hasLabelCol(schema) || !$(forceIndexLabel),
+    require(
+      !hasLabelCol(schema) || ! $(forceIndexLabel),
```

Comment:
Likewise around here I wouldn't reformat; it's pretty fine or at least consistent with the rest of the code

Reply:
reverted format changes